crossplane/aws-utils.libsonnet

```diff
@@ -129,6 +130,19 @@ local bucketPolicy = aws.s3.v1alpha3.bucketPolicy;
     },
   },
 
+ // notificationConfiguration  for S3 virus scan 
+ // https://github.com/letsbuilders/DevOps/issues/59
+ 	notificationConfiguration:: {
+    queueConfigurations: [
+      {
+        queueArn: c.aws.queueArn,
```

Comment:
I'd probably prefer if `c.aws.queueArn` was placed under `c.aws.bucket.notifications.queueArn`. Keeping it at the top level seems too broad.